### PR TITLE
Sets isKindle when inside app web view on Kindle

### PR DESCRIPTION
**Pull request by FostUK**
Sets isKindle when inside app web view on Kindle

### CODE REVIEW

File: src/core/browser.js [hunk 1]
```diff
@@ -15,7 +15,10 @@ export const getBrowser = () => {
 
     const name = browserInfo.getBrowserName();
     const version = browserInfo.getBrowserVersion();
-    const isSilk = name === "Amazon Silk";
+    const isKindleWebView = new RegExp(
+        /KFOT|KFTT|KFJWI|KFJWA|KFOTE|KFSOWI|KFTHWI|KFTHWA|KFAPWI|KFAPWA|WFJWAE|KFSAWA|KFSAWI|KFASWI|KFARWI|KFFOWI|KFGIWI|KFMEWI/g,
```
Comment (AnthonyGoodchild) [inline src/core/browser.js:19]: 🤔

Comment (AnthonyGoodchild) [reply src/core/browser.js:19]: ah android build refs or something?

**Review by AnthonyGoodchild — APPROVED** [review]
looks good just a linting fix 👍

File: src/core/browser.js (hunk 1 above)
Comment (FostUK) [reply src/core/browser.js:19]: Yeah list supplied by apps. Not sure any other ways to narrow it down exist. Was hoping could add " Build" to the end but that may or may not be present.

**Pull request merged**